.gitignore

```diff
@@ -1,5 +1,5 @@
-node_modules
-out
+/node_modules
+/out
```

Comment:
the old `out` seems out now, have to add a `/` to make new added localization .JSON files visible to the `git diff` command